data/json/items/comestibles/junkfood.json

```diff
@@ -1187,5 +1187,49 @@
     "charges": 4,
     "vitamins": [ [ "calcium", 4 ] ],
     "fun": 6
+  },
+  {
+    "type": "COMESTIBLE",
+    "id": "freeze_dried_meal",
+    "name": "freeze-dried meal",
+    "weight": "175 g",
+    "color": "red",
+    "spoils_in": "1461 days",
+    "container": "bag_plastic",
+    "comestible_type": "FOOD",
+    "symbol": "%",
+    "quench": -8,
+    "calories": 879,
+    "description": "A sealed plastic bag containing a full dehydrated meal.  This one contains a hearty cooked beef and egg breakfast.",
+    "price": 1200,
+    "price_postapoc": 1400,
+    "material": [ "flesh", "egg", "junk" ],
+    "volume": "250ml",
+    "charges": 1,
+    "flags": [ "EATEN_HOT" ],
+    "vitamins": [ [ "vitA", 3 ], [ "iron", 10 ] ],
+    "fun": -2
+  },
+  {
+    "type": "COMESTIBLE",
+    "id": "freeze_dried_meal_hydrated",
+    "name": "rehydrated freeze-dried meal",
+    "weight": "675 g",
+    "color": "brown",
+    "spoils_in": "1 day",
+    "container": "bag_plastic",
+    "comestible_type": "FOOD",
+    "symbol": "%",
+    "quench": 1,
+    "calories": 879,
+    "description": "This is a rehydrated meal meant for camping or hiking.",
+    "price": 1200,
+    "price_postapoc": 1400,
+    "material": [ "flesh", "egg", "junk" ],
+    "volume": "500ml",
+    "charges": 2,
```

Comment:
2?

Reply:
Maybe 1 would have been better. It's just so characters can control their portion sizes.

Reply:
The calories are spread between the two charges?

Reply:
I assumed that's how it worked.

Reply:
I'm not sure; this is evidently a confusing area - see #36270.

Reply:
Let's not use charges here in this particular space.  Each json entry should be for a single unit of an item.  set the charges you want it to appear in the itemgroups.

Reply:
@Maleclypse Charges have been removed.